climakitae/core.py

```diff
@@ -132,19 +132,39 @@ def load(self, data):
         return _compute(data)
 
     # === Retrieve ===================================
-    def retrieve(self):
+    def retrieve(self, config=None, merge=True):
         """Retrieve data from catalog
-
-        Applications.selections and Applications.location determine data retrieves
-        Grabs the data from the AWS S3 bucket, returns lazily loaded dask array
-        User-facing function that provides a wrapper for _read_from_catalog
-
+        
+        By default, Application.selections and Application.location determines the data retrieved. 
+        To retrieve data using the settings in a configuration csv file, set config to the local 
+        filepath of the csv.
+        Grabs the data from the AWS S3 bucket, returns lazily loaded dask array.
+        User-facing function that provides a wrapper for _read_from_catalog and _read_data_from_csv. 
+        
+        Parameters
+        ----------
+        config: str, optional. 
+            Local filepath to configuration csv file
+            Default to None-- retrieve settings in app.selections and app.location
+        merge: bool, optional
+            If config is TRUE and multiple datasets desired, merge to form a single object?
+            Defaults to True.
+        
         Returns
         -------
         xr.DataArray
             Lazily loaded dask array
 
         """
+        if config is not None:
+            if type(config) == str:
+                return _read_data_from_csv(
```

Comment:
Wishing there were symmetry between this function name and _read_from_catalog... maybe rename them:
_read_catalog_from_select
_read_catalog_from_csv
?

Reply:
yep great idea.